Dockerfile.base

```diff
@@ -1,7 +1,7 @@
-FROM nvidia/cuda:11.8.0-cudnn8-devel-ubuntu22.04
+FROM nvidia/cuda:12.4.1-cudnn-devel-ubuntu22.04
 
 ENV INSTALL_OPTIONAL=TRUE
-ENV MAX_JOBS=8
+ENV MAX_JOBS=32
```

Comment:
we're building this image with gh actions
so 32 cores may be not available

Reply:
ok